ic-exports/src/pocket_ic.rs

```diff
@@ -0,0 +1,140 @@
+use std::io::{Cursor, Read};
+use std::path::{Path, PathBuf};
+use std::time::Duration;
+use std::{env, fs};
+
+use flate2::read::GzDecoder;
+use log::*;
+use once_cell::sync::Lazy;
+pub use pocket_ic::*;
+
+#[cfg(feature = "pocket-ic-tests-async")]
+pub mod asnc;
```

Comment:
`asnc`?! is this to avoid the keyword `async`? If yes, then use `nio` instead (Non-blocking-IO)

Reply:
Done)